Framework/PythonInterface/mantid/plots/datafunctions.py

```diff
@@ -924,7 +924,7 @@ def get_errorbar_bounds(container):
         min_x = np.min(coords)
     y_segments = _get_y_errorbar_segments(container)
     if y_segments:
-        coords = [array[:, 1] for array in y_segments]
+        coords = [array[:, 1] for array in y_segments if len(array.shape) == 2]
```

Comment:
Think there is still a bug here if all the errors are `NaN` (which occurs if you run the algorithm with `NeutronPathsMultiple=1`). It produces this error when showing errorbars
```
Traceback (most recent call last):
  File "C:\mantid-conda\mantid\qt\applications\workbench\workbench\plotting\figureerrorsmanager.py", line 84, in update_plot_after
    func(*args, **kwargs)
  File "C:\mantid-conda\mantid\qt\applications\workbench\workbench\plotting\figureerrorsmanager.py", line 32, in toggle_all_errors
    cls.toggle_error_bars_for(ax, line, make_visible)
  File "C:\mantid-conda\mantid\qt\applications\workbench\workbench\plotting\figureerrorsmanager.py", line 73, in toggle_error_bars_for
    cls.update_limits_and_legend(ax, legend_props)
  File "C:\mantid-conda\mantid\qt\applications\workbench\workbench\plotting\figureerrorsmanager.py", line 101, in update_limits_and_legend
    ax.relim()
  File "C:\mantid-conda\mantid\Framework\PythonInterface\mantid\plots\mantidaxes.py", line 529, in relim
    min_x, max_x, min_y, max_y = datafunctions.get_errorbar_bounds(container)
  File "C:\mantid-conda\mantid\Framework\PythonInterface\mantid\plots\datafunctions.py", line 928, in get_errorbar_bounds
    max_y = np.max(coords)
  File "<__array_function__ internals>", line 180, in amax
  File "C:\Users\xhg73778\AppData\Local\mambaforge\envs\mantid-developer\lib\site-packages\numpy\core\fromnumeric.py", line 2793, in amax
    return _wrapreduction(a, np.maximum, 'max', axis, None, out,
  File "C:\Users\xhg73778\AppData\Local\mambaforge\envs\mantid-developer\lib\site-packages\numpy\core\fromnumeric.py", line 86, in _wrapreduction
    return ufunc.reduce(obj, axis, dtype, out, **passkwargs)
ValueError: zero-size array to reduction operation maximum which has no identity
```
Maybe there should be a test for this somewhere as well?

Reply:
I've added a test